src/Common/src/System/PasteArguments.cs

```diff
@@ -17,8 +17,9 @@ public static string Paste(IEnumerable<string> arguments, bool pasteFirstArgumen
         {
             var stringBuilder = new StringBuilder();
 
-            foreach (string argument in arguments)
```

Comment:
It's pasting together a set of strings to form a single string. Since the the class name is called "Paste**Arguments**" and it has parameter that mentions "argv0", I think the meaning is pretty clear.